src/lib/variants.test.ts

```diff
@@ -169,6 +183,52 @@ describe('variant filters', () => {
         expect(got2).toBe(false);
     });
 
+    it('should filter by user location', () => {
```

Comment:
I might have a play with table tests here and https://www.npmjs.com/package/@guardian/jest-to-be-hint but more a reminder for myself than anything else!